### PR TITLE
teku pyrmont

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

File: compose-examples/teku-only/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Runs on http://localhost:9094, scrapes data of beacon and validator.
+
+### Grafana
+Grafana listens on http://localhost:3004 and uses the data provided by prometheus service.
+
+Login with username `admin` and password `admin` (Grafana defaults), data source to Prometheus is already established and dashboards installed.
+
+## FAQ
+
+### How do I install docker and docker-compose on raspberry pi?
```
Comment (stefa2k) [inline compose-examples/teku-only/README.md:66]: please remove this bock, we don't have ARM64 builds yet

File: compose-examples/teku-only/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    options:
+      max-file: "10"
+      max-size: "100m"
+
+services:
+  geth:
+    image: ethereum/client-go:${GETH_DOCKER_TAG}
+    restart: always
+    hostname: geth
+    command: --goerli --rpc --rpcport=8545 --rpcaddr=0.0.0.0 --rpcvhosts="*" --allow-insecure-unlock --rpcapi="db,eth,net,web3,personal"
```
Comment (stefa2k) [inline compose-examples/teku-only/docker-compose.yaml:15]: use "http" instead of "rpc" like here: https://github.com/stereum-dev/ethereum2-docker-compose/blob/master/compose-examples/prysm-only/docker-compose.yaml#L15

File: config/grafana/provisioning-t/datasources/datasource.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  # <string> url
+  url: http://prometheus:9090
+  # <string> database password, if used
+  password:
+  # <string> database user, if used
+  user:
+  # <string> database name, if used
+  database:
+  # <bool> enable/disable basic auth
+  basicAuth: true
```
Comment (stefa2k) [inline config/grafana/provisioning-t/datasources/datasource.yml:29]: set basicAuth to `false`, it's not necessary

File: config/prometheus/prometheus-t.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:9094"]
+  - job_name: "beacon"
+    scrape_timeout: 10s
+    metrics_path: /metrics
+    scheme: http
+    static_configs:
+      - targets: ["beacon:8008"]
+  - job_name: "validator"
```
Comment (stefa2k) [inline config/prometheus/prometheus-t.yaml:15]: there is no `validator` service to query

File: config/prysm/beacon.yaml [hunk 5]
```diff
@@ -17,6 +17,8 @@ p2p-host-dns: ""
 rpc-host: 0.0.0.0
 monitoring-host: 0.0.0.0
 
+grpc-gateway-corsdomain: http://beacon-chain:7500
```
Comment (stefa2k) [inline config/prysm/beacon.yaml:20]: I guess this doesn't belong in this pull request, right? It's for prysm

**Review by stefa2k — CHANGES_REQUESTED** [review]
Please review the comments, also there is `data/prysm/validator/db/validator.db` in this pull request, shouldn't be included.

File: compose-examples/teku-only/docker-compose.yaml (hunk 2 above)
Comment (gbayasgalan) [reply compose-examples/teku-only/docker-compose.yaml:15]: okey

File: compose-examples/teku-only/README.md (hunk 1 above)
Comment (gbayasgalan) [reply compose-examples/teku-only/README.md:66]: okey, got it

File: config/prysm/beacon.yaml (hunk 5 above)
Comment (gbayasgalan) [reply config/prysm/beacon.yaml:20]: yes, it is for prysm

**Review by stefa2k — APPROVED** [review]
lgtm
